src/Raven.Server/Rachis/ClusterNodeStatusReport.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using Raven.Client.Documents.Replication.Messages;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Rachis
+{
+    public enum DatabaseStatus
+    {
+        Loaded,
+        Loading,
+        Faulted,
+        Unloaded,
+    }
+    
+    public class DatabaseStatusReport : IDynamicJson
+    {
+        public string Name;
+        public string NodeName;
+
+        public ChangeVectorEntry[] LastDocumentChangeVector;
+
+        public ChangeVectorEntry[] LastAttachmentChangeVector;
+
+        // <collection,etag>
+        public Dictionary<string,long> LastIndexedDocumentEtag = new Dictionary<string, long>();
```

Comment:
This make no sense, there are multiple indexes that are running, each at their own pace. 
So this is not valid thing to say.